features/get-dataset-json.feature

```diff
@@ -476,15 +476,12 @@ Feature: Get Dataset JSON
           "dimension_name":"city",
           "options":[
             {
-              "href":"http://hostname/code-lists/city/codes/0",
               "id":"0"
```

Comment:
How was this working before? Doesn't look like you changed this in the code :/

Reply:
It was removed in this PR as per the ticket - https://github.com/ONSdigital/dp-cantabular-filter-flex-api/pull/159 

but I removed the whole Options field rather than just the href by mistake, so ID needed to be reinstated